ElGrecode/aaa-component-library
# Theme Provider with Breakpoints base case

### Links
Code Complete: https://autoclub.atlassian.net/browse/POB-5877
Resolves https://autoclub.atlassian.net/browse/POB-5882
Resolves https://autoclub.atlassian.net/browse/POB-5884

### Description
_Detail with what was changed, why it was changed, and how it was changed_ 
--**The problem**:  
Needed to detail the use of themes and create a base case for palette and breakpoints

### How to test
`yarn storybook` to run the storybook and check out `Button.js` and `AAAThemeProvider.js` to see the use of themes

### Additional Notes
I will leave commentary in the body

### Explanation of Jest/Unit Testing
No new test cases
